cmd/preflight/cmd/check_container_test.go

```diff
@@ -145,24 +152,24 @@ var _ = Describe("Check Container Command", func() {
 		Entry("index manifest, invalid platform", "index", "none", HaveOccurred(), true),
 	)
 
-	Context("When validating check container arguments and flags", func() {
+	When("validating check container arguments and flags", func() {
```

Comment:
Non-blocking: Why did you switch from `context` to `when`, is there some benefit? I'm fine with the change, but just want to understand the driving factor.

Reply:
It does the same thing, but "reads" better. In both the code and the output when a test fails.